File: Ecommerce-SPA-Admin/src/app/_core/models/product.ts
export interface Product {
    id: number;
    code: string;
    title: string;
    description: string;
    avartar: string;
    thumb: string;
    content: string;
    imageList: string;
    price: number;
    oldPrice: number | null;
    promotion: string;
    isHot: boolean | null;
    isSale: boolean | null;
    isNew: boolean | null;
    quantity: number;
    status: boolean | null;
    productCategoryId: number;
    createBy: string;
    createTime: string;
    updateBy: string;
    updateTime: string | null;
}

File: Ecommerce-API/Models/Product.cs

using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Ecommerce_API.Models
{
    public partial class Product
    {
        [Key]
        public int Id { get; set; }
        [StringLength(50)]
        public string Code { get; set; }
        [StringLength(250)]
        public string Title { get; set; }
        public string Description { get; set; }
        [StringLength(250)]
        public string Avartar { get; set; }
        [StringLength(250)]
        public string Thumb { get; set; }
        public string Content { get; set; }
        public string ImageList { get; set; }
        public double Price { get; set; }
        public double? OldPrice { get; set; }
        public string Promotion { get; set; }
        public bool? IsHot { get; set; }
        public bool? IsSale { get; set; }
        public bool? IsNew { get; set; }
        public int Quantity { get; set; }
        public bool? Status { get; set; }
        public int ProductCategoryId { get; set; }
        [Required]
        [StringLength(50)]
        public string CreateBy { get; set; }
        [Column(TypeName = "datetime")]
        public DateTime CreateTime { get; set; }
        [StringLength(50)]
        public string UpdateBy { get; set; }
        [Column(TypeName = "datetime")]
        public DateTime? UpdateTime { get; set; }
    }
}
